<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qync::MainWindow</class>
 <widget class="QMainWindow" name="Qync::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="presets"/>
    </item>
    <item>
     <widget class="QTabWidget" name="settingsTab">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="basicSettings">
       <attribute name="icon">
        <iconset theme="preferences-other">
         <normaloff>.</normaloff>.</iconset>
       </attribute>
       <attribute name="title">
        <string>Basic settings</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QGridLayout" name="sourceAndDestinationLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="sourceLabel">
            <property name="text">
             <string>Source</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="source"/>
          </item>
          <item row="0" column="2">
           <widget class="QToolButton" name="chooseSource">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset theme="document-open">
              <normaloff>.</normaloff>.</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="destinationLabel">
            <property name="text">
             <string>Destination</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="destination"/>
          </item>
          <item row="1" column="2">
           <widget class="QToolButton" name="chooseDestination">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset theme="document-open">
              <normaloff>.</normaloff>.</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="preserveBox">
          <property name="title">
           <string>Preserve</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QCheckBox" name="preserveTime">
             <property name="text">
              <string>Time</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="preserveOwner">
             <property name="text">
              <string>Owner</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="preservePermissions">
             <property name="text">
              <string>Permissions</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="preserveGroup">
             <property name="text">
              <string>Group</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="otherSettingsBox">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QCheckBox" name="windowsCompatible">
             <property name="text">
              <string>Windows compatible</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="honourDeletions">
             <property name="text">
              <string>Honour deletions</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="basicSettingsSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="advancedSettings">
       <attribute name="icon">
        <iconset theme="preferences-system">
         <normaloff>.</normaloff>.</iconset>
       </attribute>
       <attribute name="title">
        <string>Advanced settings</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QGridLayout" name="advancedSettingsGrid">
          <item row="0" column="0">
           <widget class="QCheckBox" name="alwaysCompareChecksums">
            <property name="text">
             <string>Always compare checksums</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="compressInTransit">
            <property name="text">
             <string>Compress files in transit</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="preserveDevices">
            <property name="text">
             <string>Preserve devices</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="onlyUpdateExisting">
            <property name="text">
             <string>Only update existing files</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="keepPartialFiles">
            <property name="text">
             <string>Keep partially transferred files</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="dontMapUidGid">
            <property name="text">
             <string>Don't map UID/GID values</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="symlinksAsSymlinks">
            <property name="text">
             <string>Copy symlinks as synlinks</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="hardlinksAsHardlinks">
            <property name="text">
             <string>Copy hardlinks as hardlinks</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="makeBackups">
            <property name="text">
             <string>Make backups</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QCheckBox" name="itemisedChanges">
            <property name="text">
             <string>Show itemised changes</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="logFileLayout">
          <item>
           <widget class="QLabel" name="logFileLabel">
            <property name="text">
             <string>Log file</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="logFile"/>
          </item>
          <item>
           <widget class="QToolButton" name="chooseLogFile">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset theme="document-open">
              <normaloff>.</normaloff>.</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="advancedSettingsSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>135</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>708</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionChooseSource"/>
    <addaction name="actionChooseDestination"/>
    <addaction name="actionSwitchSourceDestination"/>
    <addaction name="separator"/>
    <addaction name="actionSimulate"/>
    <addaction name="actionSync"/>
    <addaction name="separator"/>
    <addaction name="actionPreferences"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuPresets">
    <property name="title">
     <string>Presets</string>
    </property>
    <widget class="QMenu" name="menuMyPresets">
     <property name="title">
      <string>My presets</string>
     </property>
    </widget>
    <addaction name="separator"/>
    <addaction name="menuMyPresets"/>
    <addaction name="separator"/>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionRemove"/>
    <addaction name="separator"/>
    <addaction name="actionExport"/>
    <addaction name="actionImport"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutRsync"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPresets"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="presetsToolbar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="actionRemove"/>
  </widget>
  <widget class="QToolBar" name="synchroniseToolbar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSimulate"/>
   <addaction name="actionSync"/>
  </widget>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about" resource="../resources/icons.qrc">
     <normaloff>:/icons/menu/help/about</normaloff>:/icons/menu/help/about</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAboutRsync">
   <property name="text">
    <string>About rsync</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset theme="document-save-as">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset theme="list-remove" resource="../resources/icons.qrc">
     <normaloff>:/icons/menu/presets/remove</normaloff>:/icons/menu/presets/remove</iconset>
   </property>
   <property name="text">
    <string>Remove</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionChooseSource">
   <property name="icon">
    <iconset theme="folder" resource="../resources/icons.qrc">
     <normaloff>:/icons/menu/file/choosedir</normaloff>:/icons/menu/file/choosedir</iconset>
   </property>
   <property name="text">
    <string>Choose source folder</string>
   </property>
  </action>
  <action name="actionChooseDestination">
   <property name="icon">
    <iconset theme="folder-red" resource="../resources/icons.qrc">
     <normaloff>:/icons/menu/file/choosedir</normaloff>:/icons/menu/file/choosedir</iconset>
   </property>
   <property name="text">
    <string>Choose destination folder</string>
   </property>
  </action>
  <action name="actionSwitchSourceDestination">
   <property name="text">
    <string>Switch source and destination</string>
   </property>
  </action>
  <action name="actionSimulate">
   <property name="icon">
    <iconset theme="document-edit-verify" resource="../resources/icons.qrc">
     <normaloff>:/icons/menu/file/simulate</normaloff>:/icons/menu/file/simulate</iconset>
   </property>
   <property name="text">
    <string>Simulate</string>
   </property>
  </action>
  <action name="actionSync">
   <property name="icon">
    <iconset theme="media-playback-start" resource="../resources/icons.qrc">
     <normaloff>:/icons/menu/file/execute</normaloff>:/icons/menu/file/execute</iconset>
   </property>
   <property name="text">
    <string>Synchronise</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset theme="preferences-system">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="application-exit" resource="../resources/icons.qrc">
     <normaloff>:/icons/menu/file/quit</normaloff>:/icons/menu/file/quit</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset theme="document-new" resource="../resources/icons.qrc">
     <normaloff>:/icons/toolbar/new</normaloff>:/icons/toolbar/new</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>presets</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Qync::MainWindow</receiver>
   <slot>showPreset(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>708</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chooseLogFile</sender>
   <signal>clicked()</signal>
   <receiver>Qync::MainWindow</receiver>
   <slot>chooseLogFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>683</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>706</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chooseSource</sender>
   <signal>clicked()</signal>
   <receiver>Qync::MainWindow</receiver>
   <slot>chooseSourceDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>678</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>707</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chooseDestination</sender>
   <signal>clicked()</signal>
   <receiver>Qync::MainWindow</receiver>
   <slot>chooseDestinationDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>684</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>708</x>
     <y>188</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>Qync::MainWindow</receiver>
   <slot>about()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAboutRsync</sender>
   <signal>triggered()</signal>
   <receiver>Qync::MainWindow</receiver>
   <slot>aboutRsync()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionChooseDestination</sender>
   <signal>triggered()</signal>
   <receiver>Qync::MainWindow</receiver>
   <slot>chooseDestinationDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionChooseSource</sender>
   <signal>triggered()</signal>
   <receiver>Qync::MainWindow</receiver>
   <slot>chooseSourceDirectory()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExport</sender>
   <signal>triggered()</signal>
   <receiver>Qync::MainWindow</receiver>
   <slot>exportPreset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionImport</sender>
   <signal>triggered()</signal>
   <receiver>Qync::MainWindow</receiver>
   <slot>importPreset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNew</sender>
   <signal>triggered()</signal>
   <receiver>Qync::MainWindow</receiver>
   <slot>addNewPreset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPreferences</sender>
   <signal>triggered()</signal>
   <receiver>Qync::MainWindow</receiver>
   <slot>showPreferences()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>Qync::MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRemove</sender>
   <signal>triggered()</signal>
   <receiver>Qync::MainWindow</receiver>
   <slot>removePreset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>Qync::MainWindow</receiver>
   <slot>saveSettingsToCurrentPreset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSaveAs</sender>
   <signal>triggered()</signal>
   <receiver>Qync::MainWindow</receiver>
   <slot>newPresetFromSettings()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSimulate</sender>
   <signal>triggered()</signal>
   <receiver>Qync::MainWindow</receiver>
   <slot>simulate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSwitchSourceDestination</sender>
   <signal>triggered()</signal>
   <receiver>Qync::MainWindow</receiver>
   <slot>switchSourceAndDestination()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSync</sender>
   <signal>triggered()</signal>
   <receiver>Qync::MainWindow</receiver>
   <slot>synchronise()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showPreset(int)</slot>
  <slot>chooseSourceDirectory()</slot>
  <slot>chooseDestinationDirectory()</slot>
  <slot>chooseSourceFile()</slot>
  <slot>chooseDestinationFile()</slot>
  <slot>chooseLogFile()</slot>
  <slot>switchSourceAndDestination()</slot>
  <slot>showPreferences()</slot>
  <slot>importPreset()</slot>
  <slot>exportPreset()</slot>
  <slot>about()</slot>
  <slot>aboutQt()</slot>
  <slot>aboutRsync()</slot>
  <slot>addNewPreset()</slot>
  <slot>removeCurrentPreset()</slot>
  <slot>saveSettingsToCurrentPreset()</slot>
  <slot>newPresetFromSettings()</slot>
  <slot>simulate()</slot>
  <slot>synchronise()</slot>
 </slots>
</ui>
